requirements/requirements_nemo.txt

```diff
@@ -1,6 +1,7 @@
-git+https://github.com/NVIDIA/NeMo.git@d0efff087613ea2584e215969f289fed17414d8b#egg=nemo_toolkit[all] # This commit hash is a recent version of main at the time of testing.
+git+https://github.com/NVIDIA/NeMo.git@cb755f5595880a56159cc9a6f4a050c20a449d0a#egg=nemo_toolkit[all] # This commit hash is a recent version of main at the time of testing.
 tqdm
 soundfile
 librosa
 IPython # Workaround for https://github.com/NVIDIA/NeMo/pull/9890#discussion_r1701028427
 cuda-python>=12.4 # Used for fast TDT and RNN-T inference
+datasets <= 2.21.0
```

Comment:
this is present in general requirements. so not required

Reply:
Added the upper bound here as there have been some breaking changes in datasets 3.x.x, however removed this in the updated version of the script and handled both versions in code